app/build.gradle.kts

```diff
@@ -38,11 +38,6 @@ android {
 	composeOptions {
 		kotlinCompilerExtensionVersion = "1.4.7"
 	}
-	packagingOptions {
```

Comment:
It can be that I wasn't at the guild when it was discussed, but why do we want to include those directories in our APK/AAB?

Reply:
I'm wondering what was the purpose of excluding them?

Reply:
Fully agree but in terms of redundant configuration, `packagingOptions` in this case. We don't want to struggle with future migrating or maintaining, parts of code, which are not needed